cosmoz-grouped-list.js

```diff
@@ -215,11 +214,12 @@
 			if (!data[0].items) {
 				// no grouping, so render items as a standard list
 				this._groupsMap = null;
+				this.selectedItems = this.selectedItems.filter(i => data.indexOf(i) >= 0);
```

Comment:
Can we also add a test that covers this line please?
```suggestion
				this.selectedItems = this.selectedItems.filter(i => data.includes(i));
```